3 - PB/Documentazione esterna/Piano di Qualifica/Piano di Qualifica.typ

```diff
@@ -432,127 +432,68 @@ In questa sezione sono elencati i test eseguiti sul prodotto che, come riportato
-      [I-5.3],[*productRepository*: verifica che venga correttamente gestito un errore quando si vogliono restituire tutti i prodotti],[P],
-      [I-5.4],[*productRepository*: verifica che venga correttamente gestito un errore quando si vuole restituire un prodotto cercato per ID],[P],
-      [I-5.5],[*productRepository*: verifica che venga correttamente restituito un array vuoto quando non viene trovato nessun prodotto],[P],
-      [I-5.6],[*productRepository*: verifica che venga correttamente restituito null quando non vengono trovati prodotti cercando per ID],[P],
-      [I-6.1],[*zoneRepository*: verifica che vengano correttamente restituite tutte le zone],[P],
-      [I-6.2],[*zoneRepository*: verifica che vengano correttamente restituite tutte le zone che non contengono prodotti],[P],
-      [I-6.3],[*zoneRepository*: verifica che venga correttamente restituita la zona cercata per ID], [P],
-      [I-6.4],[*zoneRepository*: verifica che venga correttamente gestito un errore quando si vogliono restituire tutte le zone],[P],
-      [I-6.5],[*zoneRepository*: verifica che venga correttamente gestito un errore quando si vuole restituire una zona cercata per ID],[P],
-      [I-6.6],[*zoneRepository*: verifica che venga correttamente restituito un array vuoto quando non vengono trovate zone],[P],
-      [I-6.7],[*zoneRepository*: verifica che venga correttamente restituito null quando non viene trovata la zona cercata per ID],[P],
-      [I-7.1],[*readSavedSVG*: verifica che venga correttamente letto il contenuto del file SVG],[P],
-      [I-8.1],[*readSavedSVG*: verifica che venga correttamente salvato il contenuto del file SVG],[P],
-      [I-9.1],[*SVGSanitizer*: verifica che venga correttamente sanificato il file SVG di esempio],[P],
-      [I-9.2],[*SVGSanitizer*: verifica che venga correttamente sanificato il file SVG fornito da Sanmarco Informatica],[P],
-      [I-10.1],[*getAllEmptyZones*: verifica che vengano correttamente ritornate tutte le zone vuote],[P],
-      [I-11.1],[*getAllZones*: verifica che vengano correttamente ritornate tutte le zone],[P],
-      [I-12.1],[*getBinsByZoneId*: verifica che vengano correttamente ritornati tutti i bin dato l'ID di una zona],[P],
-      [I-13.1],[*getZoneById*: verifica che venga correttamente ritornata la zona cercata per ID e i suoi bin],[P],
-  ),
-  caption: "Tabella integration test - risultato"
+  table-json(json("integrationTest.json")),
+  caption: "Tabella integration test"
 )
 
+== Test di sistema
+La suite di test di sistema ha lo scopo di verificare che il sistema soddisfi i requisiti definiti nel documento #adr_v. L'implementazione di test automatici per la parte interattiva del prodotto e per l'ambiente tridimensionale è stata ritenuta eccessivamente complessa in termini di tempo e risorse valutando l'inesperienza del gruppo. Non risultava però ragionevole rinunciare a questa tipologia di test, pertanto si è deciso di svolgerli manualmente.
+
```

Comment:
Aggiungere `#show "<=": [$<=$]` in modo da visualizzare correttamente il simbolo

Reply:
Grazie per la segnalazione. Lo metterò all'interno del file json, in quanto la tabella che viene creata è in grado di eseguire le direttive di markup